karate-core/src/main/java/com/intuit/karate/driver/DriverOptions.java

```diff
@@ -167,12 +169,20 @@ public DriverOptions(ScenarioContext context, Map<String, Object> options, LogAp
                 args.add(executable);
             }
         }
-        workingDir = new File(FileUtils.getBuildDir() + File.separator + uniqueName);
-        if (options.containsKey("userDataDir")) { // special case allow user-specified null
-            userDataDir = (String) options.get("userDataDir");
+        disableNotifications = get("disableNotifications", false);
+        userAgent = get("userAgent", null);
+        String place = get("userDataDir", null);
```

Comment:
I wanted user to provide `null` in which case chrome would use system defaults
and not providing would default to workingDir

will this work the same way

Reply:
Leaving workingDir like that, will create countless 'chrome_1234567890' folders inside target with zero content.<br/>
Beacause Command class will check if it exists and create one if not.<br/>
This way, workingDir and userDataDir are the same.